src/Components/Contact/ContactForm.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react'
+import { Formik, Form, Field, ErrorMessage } from 'formik'
+
+import '../FormStyles.css'
+
+const ContactForm = () => {
+  return (
+    <>
+      <Formik
+        initialValues={{
+          name: '',
+          email: '',
+          phone: '',
+          message: '',
+        }}
+        validate={(values) => {
+          let errors = {}
+          if (!values.name) {
+            errors.name = 'Debes poner tu nombre'
+          }
+
+          if (!values.email) {
+            errors.email = 'Debes poner tu email'
+          } else if (
+            !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(values.email)
```

Comment:
what's the meaning of this regex? yo make the code more legible please create a constant or function that in its name says what it means. For example:
```
const isInvalidEmail = (email) => {
    const emailRegexValidation = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
    return !emailRegexValidation.test(email);
}
... code

const validations = (contact) => {
    validations ....
    if(!contact.email) errors.email = "el email es requerido";
    else if(isInvalidEmail(contact.email) errors.email = "La dirección de Email no es valida";
...rest of the code
```

Reply:
Ok. Ready!